src/cascadia/TerminalApp/ActionAndArgs.cpp

```diff
@@ -274,6 +282,8 @@ namespace winrt::TerminalApp::implementation
                 { ShortcutAction::RenameTab, RS_(L"ResetTabNameCommandKey") },
                 { ShortcutAction::ExecuteCommandline, RS_(L"ExecuteCommandlineCommandKey") },
                 { ShortcutAction::ToggleCommandPalette, RS_(L"ToggleCommandPaletteCommandKey") },
+                { ShortcutAction::CloseOtherTabs, L"" },
```

Comment:
Leave comment that these are blank (unlike the others) because of the generate function.